src/CitizenProject/CitizenProjectManager.php

```diff
@@ -483,7 +484,19 @@ public function approveCommitteeSupport(Committee $committee, CitizenProject $ci
         $committeeSupport->approve();
 
         if ($flush) {
-            $this->getManager()->persist($committeeSupport);
+            $this->getManager()->flush();
+        }
+    }
+
+    public function deleteCommitteeSupport(Committee $committee, bool $flush = true): void
+    {
+        if (!$committeeSupport = $this->getCitizenProjectCommitteeSupportRepository()->findByCommittee($committee)) {
+            throw new CitizenProjectCommitteeSupportNotFoundException($committee);
```

Comment:
>I create this exception to precise exception type and catch it more simply, I use this exception in deleteCommitteeSupport on Manager

Yes my point is that there is no specific catch for it, and this could be a more generic exception, I don't see the value of using a domain one here for now.